src/main/java/org/folio/consortia/client/UserCapabilitiesClient.java

```diff
@@ -1,19 +1,13 @@
 package org.folio.consortia.client;
 
-import org.folio.consortia.domain.dto.UserCapabilitiesRequest;
 import org.folio.spring.config.FeignClientConfiguration;
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PutMapping;
-import org.springframework.web.bind.annotation.RequestBody;
 
 @FeignClient(name = "user-capabilities-client", url = "users", configuration = FeignClientConfiguration.class)
 public interface UserCapabilitiesClient {
 
-  @PutMapping("/{userId}/capabilities")
-  void assignUserCapabilities(@PathVariable("userId") String userId, @RequestBody UserCapabilitiesRequest request);
-
   @DeleteMapping("/{userId}/capabilities")
   void deleteUserCapabilities(@PathVariable("userId") String userId);
```

Comment:
This method is used to delete capabilities for the shadow users when the primary user was deleted. Need to create new story to extend this logic and delete related user-role and user-capability-sets associations